api/src/main/java/org/pmiops/workbench/api/CohortAnnotationDefinitionController.java

```diff
@@ -82,6 +84,12 @@ public CohortAnnotationDefinition apply(org.pmiops.workbench.db.model.CohortAnno
         this.workspaceService = workspaceService;
     }
 
+    private void validateColumnName(String columnName) {
+        if (AnnotationQueryBuilder.RESERVED_COLUMNS.contains(columnName)) {
```

Comment:
nit: reduce indent?

Reply:
My intellij is set to 4 indent. I'm fine with either. Let me know and i can change it if need be.

Reply:
OK, then let's go with 2-space indentation; it makes the lines shorter which makes it easier to review code and reduces the need for wrapping.

At some point (in a separate PR) we should probably re-indent existing code so it's consistent.